app/common/services/strataService.js

```diff
@@ -461,6 +461,37 @@ angular.module('RedhatAccess.common').factory('strataService', [
                         return deferred.promise;
                     }
                 },
+                externalUpdates: {
+                    list: function (id) {
+                        var deferred = $q.defer();
+                        if (!ie8 && strataCache.get('externalUpdates' + id)) {
+                            //Changing cache response. Making sortModifiedDate as Date before sending
+                            var externalUpdates=strataCache.get('externalUpdates' + id);
+                            angular.forEach(externalUpdates, angular.bind(this, function (externalUpdates) {
+                                externalUpdates.sortModifiedDate=new Date(externalUpdates.sortModifiedDate);
+                            }));
+
+                            deferred.resolve(externalUpdates);
+                        } else {
+                            strata.cases.externalUpdates.list(id, function (response) {
+                                angular.forEach(response , angular.bind(this, function (externalUpdate) {
+                                    var sortPublishedDate=externalUpdate.created_date;
+                                    externalUpdate.sortModifiedDate=sortPublishedDate;
+
+                                    var createdDate=RHAUtils.convertToTimezone(externalUpdate.created_date);
+                                    externalUpdate.created_date=RHAUtils.formatDate(createdDate,'MMM DD YYYY');
```

Comment:
@vrathee  Can not use single date filter as the current version of angular(1.2.1) does not support timezone filtering(formatting of 'Z'), we want timezone to be displayed as +05:30 and date filter shows this as +0530. This formatting is available from angular version 1.3.0. We can use this filter in PCM once we upgrade our angular version.